app/src/main/java/ca/llamabagel/transpo/ui/search/SearchViewModelFactory.kt

```diff
@@ -6,16 +6,17 @@ package ca.llamabagel.transpo.ui.search
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
+import ca.llamabagel.transpo.data.SearchRepository
 import javax.inject.Inject
 
-class SearchViewModelFactory @Inject constructor() : ViewModelProvider.Factory {
+class SearchViewModelFactory @Inject constructor(private val repository: SearchRepository) : ViewModelProvider.Factory {
```

Comment:
This factory should be removed and replaced with a `ViewModelFactory` in `CoreComponent`. See the other View Models as a reference.